conda/meta.yaml

```diff
@@ -21,6 +21,9 @@ requirements:
 
   run:
     - python >=3.6
+    - six==1.16.0
+    - globus-sdk==2.0.1
+    - fair-research-login==0.2.0
```

Comment:
```suggestion
    - six
    - globus-sdk
    - fair-research-login
```

Reply:
No pinning in `meta.yaml` to increase compatibility when installing in conda envs.